VPCBuilder/README.md

```diff
@@ -0,0 +1,131 @@
+# VPC Builder
+
+Builds out a "fully" featured VPC summarising the complexity associated with a VPC such as Internet & Customer Gateways, Subnets, Routetables and NATGateways.
+
+It also adds in VPC Flowlogs with an IAM role and supports full dynamic allocation of IPv6 with the VPC and to each subnet. 
+
+The IPv6 handles Egress Internet Gateway and default route against ::/0
+
+## Build Package
+
+make buildPackage
+
+## Upload package to S3
+Fill in your bucket and profile (utilises a crud aws cli s3 upload command)
+
+make uploadToS3
+
+## To Do
+
+Add Outputs with Exports for critical resources
+VPC Endpoints for all AWS Services
+Add a little better handling of custom pieces (e.g. different route gateways)
+Adding proper IPv6 regex and handling with NetworkACLs
+
+## Basic Usage
+
+Utilise the yaml structure below e a template, changing the Account ID in the transformation definiton.
```

Comment:
"below e a template" looks like another typo, presumably "below is a template".

Reply:
Corrected. Cheers.